### DISCUSSION TITLE
Is there any slack or discord community for node-saml and passport-saml?

### DISCUSSION THREAD

**Discussion by kriss1897** [node-saml/node-saml#8] [General]
@cjbarth ??

**Comment by kriss1897** [comment]
Communicating on github alone seems really challanging when I think about planning the road ahead for this repo.

**Reply by cjbarth** [reply]
There is a Slack community, but, honestly, most of the planning and discussion takes place right here. We mostly use the Slack community for discussing security-related things (responsible disclosure, and the like). What kind of discussion about the road map are you thinking of?
